src/helm/benchmark/presentation/run_display.py

```diff
@@ -0,0 +1,267 @@
+from collections import defaultdict
+from dataclasses import dataclass
+import os
+import json
+from typing import Any, Dict, List, Optional, Set, Tuple, cast
+
+import dacite
+
+from helm.benchmark.adapter import AdapterSpec, RequestState, ScenarioState
+from helm.benchmark.augmentations.dialect_perturbation import DialectPerturbation
+from helm.benchmark.augmentations.extra_space_perturbation import ExtraSpacePerturbation
+from helm.benchmark.augmentations.filler_words_perturbation import FillerWordsPerturbation
+from helm.benchmark.augmentations.gender_perturbation import GenderPerturbation
+from helm.benchmark.augmentations.misspelling_perturbation import MisspellingPerturbation
+from helm.benchmark.augmentations.person_name_perturbation import PersonNamePerturbation
+from helm.benchmark.augmentations.perturbation_description import PerturbationDescription
+from helm.benchmark.augmentations.space_perturbation import SpacePerturbation
+from helm.benchmark.augmentations.synonym_perturbation import SynonymPerturbation
+from helm.benchmark.augmentations.typos_perturbation import TyposPerturbation
+from helm.benchmark.metrics.metric import PerInstanceStats
+from helm.benchmark.presentation.schema import Schema
+from helm.benchmark.runner import RunSpec
+from helm.common.general import write, asdict_without_nones
+from helm.common.hierarchical_logger import htrack
+from helm.common.request import Request
+
+
+_PERTURBATION_NAME_TO_DESCRIPTION = {
```

Comment:
Is there a less manual way of doing this?  How come we didn't need this before?

Reply:
We can use subclass registration, but I did not do it in this PR because I wanted to avoid scope creep. I filed #1251 for that.